package-managers/nuget.md

```diff
@@ -6,10 +6,63 @@ TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-n
 
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
+### Find our feed URL
+
+#### Private feed
+
+Private feeds are only accessible to the feed owner, and to whomever the feed owner granted sufficient privileges. When we need to connect to a private feed we don't own, we may need to ask the feed owner to invite us to the feed.

```

Comment:
`and to whomever the feed owner granted sufficient privileges` -> `and to other users who were granted access by the feed owner` ? (reads a bit more fluent I think)